tracing-subscriber/src/subscribe_old.rs

```diff
@@ -213,6 +213,37 @@ where
     C: Collect,
     Self: 'static,
 {
+    /// Performs late initialization when attaching a `Subscribe` to a
```

Comment:
I think this file needs to be removed: it's not used anywhere.

Reply:
ah, whoops

Reply:
i forgot to remove it earlier, sorry!